.github/templates/main.yml

```diff
@@ -74,12 +74,9 @@ steps:
 #@ end
 
 #@ def buildPackages():
-#@ finalList = []
 #@ for pkgName in nugetPackages:
-#@  finalList.append(msbuild("Realm/" + pkgName, target="Pack", PackageOutputPath="${{ github.workspace }}/Realm/packages", VersionSuffix="${{ steps.set-version-suffix.outputs.build_suffix }}"))
+  - #@ msbuild("Realm/" + pkgName, target="Pack", PackageOutputPath="${{ github.workspace }}/Realm/packages", VersionSuffix="${{ steps.set-version-suffix.outputs.build_suffix }}")
```

Comment:
Now I understand what you meant in your comment last time. This looks better than having the array. Thanks.